sites/shahid.mbc.net/shahid.mbc.net.config.js

```diff
@@ -34,24 +34,28 @@ module.exports = {
   async channels({ lang = 'en' }) {
     const axios = require('axios')
     const items = []
-    let page = 0
-    while (true) {
-      const result = await axios
-        .get(
-          `https://api2.shahid.net/proxy/v2.1/product/filter?filter=%7B"pageNumber":${page},"pageSize":100,"productType":"LIVESTREAM","productSubType":"LIVE_CHANNEL"%7D&country=SA&language=${lang}&Accept-Language=${lang}`
-        )
-        .then(response => response.data)
-        .catch(console.error)
-      if (result.productList) {
-        items.push(...result.productList.products)
-        if (result.productList.hasMore) {
-          page++
-          continue
+    const countryCodes = ['EG', 'SA', 'US']
```

Comment:
A best practice in my opinion would have been to change the parameters that "channels" get in order to inject the country codes of our choice.


```suggestion
      async channels({ lang = 'en', country })
```

This way, you can do 

```
npm run channels:parse --- --config=./sites/shahid.mbc.net/shahid.mbc.net.config.js --output=./sites/shahid.mbc.net/shahid.mbc.net_eg-en.channels.xml --set=country:eg --set=lang:en
````